src/main/kotlin/com/sourcegraph/cody/ui/TransparentButton.kt

```diff
@@ -28,14 +31,22 @@ class TransparentButton(text: String) : JButton(text) {
   override fun paintComponent(g: Graphics) {
     UISettings.setupAntialiasing(g)
     val g2 = g.create() as Graphics2D
-    g2.composite = AlphaComposite.SrcOver.derive(opacity)
+
+    if (isEnabled) {
+      g2.composite = AlphaComposite.SrcOver.derive(enabledOpacity)
+      g.color = enabledTextColor
+    } else {
+      g2.composite = AlphaComposite.SrcOver.derive(disabledOpacity)
+      g.color = disabledTextColor
+    }
+
```

Comment:
Can you please elaborate on this condition? Can't we just hardcode that transparency is always "on"?

Reply:
Transparency is always on. This condition only changes the styling a bit b/w the two states of the button (enabled/disabled)